transactions.sql

```diff
@@ -0,0 +1,25 @@
+/* This file handles transactions on the database */
+/*1st Transaction UPDATE & ROLLBACK*/
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+
+/* 2nd Transaction UPDATE & COMMIT */
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species is NULL;
+COMMIT
+
+/*3rd Transaction DELETE & ROLLBACK*/
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+/*4th Transaction DELETE, SAVEPOINT,UPDATE,ROLLBACK,COMMIT*/
+BEGIN;
+DELETE FROM animals WHERE date_of_birth >'2022-1-1';
+SAVEPOINT delete_data;
+UPDATE animals SET weight_kg= -1*weight_kg;
+ROLLBACK TO delete_data;
+UPDATE animals SET weight_kg= -1*weight_kg WHERE weight_kg < 0;
+COMMIT;
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/154679087-06bb9ba1-c02d-49e1-b6dc-db8e039da214.png)

Therefore:

- [ ] Please, verify each of your changes.  Verify means a SELECT statement.

Example:

```SQL
BEGIN; -- start a transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
```

Please, verify your data in this way every time the requirements ask for it.

That way, you can check how the data behaves 💪 

- [ ] According to the template, this file `transactions.sql` shouldn't exist.  Please, kindly move all the content from this file to the appropriate file from the template, and then delete this file.